src/npc.cpp

```diff
@@ -1882,16 +1941,25 @@ int npc::follow_distance() const
  return 4; // TODO: Modify based on bravery, weapon wielded, etc.
 }
 
-int npc::speed_estimate(int speed) const
+int npc::speed_estimate( const Creature *what ) const
```

Comment:
Why a pointer and not a reference? Is this ever called on purpose with a possible null pointer?